docs-new/docs/reference/api-reference/index.md

```diff
@@ -23,17 +23,13 @@ Use `kubectl` to inspect the current contents of any Keptn resource:
    For example, to list all the `KeptnApp` resources
    in the `namespace1` namespace, the command is:
 
-   ```shell
-   kubectl get keptnapps -n namespace1
-   ```
+       kubectl get keptnapps -n namespace1
```

Comment:
We need to move this to a code block.

Preview with the current changes:

![Screenshot 2024-01-04 174717](https://github.com/keptn/lifecycle-toolkit/assets/98955085/aff72574-0cfb-41c9-8f2b-2e5209b926e0)

Reply:
good catch, thanks - embedding code block in backticks seemed to not work properly when used within ordered lists, so I had to use the indentation syntax for creating code blocks here - see https://daringfireball.net/projects/markdown/syntax#precode (ReadTheDocs refers to this site for recommendations on syntax) for more context